### ISSUE TITLE
Add bakery to the category list

### ISSUE THREAD

**Issue by Altonss** [organicmaps/organicmaps#3111]
I think this is a very important category and should be in the category list :)

**Comment by janusbenissa** [comment]
If you put every shop into the list than the list becomes to long.
So I use the search field if I'm looking for a bakery :-)

**Comment by Altonss** [comment]
> If you put every shop into the list than the list becomes to long. So I use the search field if I'm looking for a bakery :-)

Yes I agree the list should not become too long, but bakery is pretty essential shop I think :) Currently the list also contains "Shopping, Nocturn life, vacations with family, ..." which is also useful but I don't think a bakery is less useful for users ^^

**Comment by TheAdventurer64** [comment]
There's already a bakery category though:![image](https://user-images.githubusercontent.com/50059322/183309413-aec25d2b-a25a-4bbd-a7f4-de3496a20ad9.png)

**Comment by Altonss** [comment]
> There's already a bakery category though:![image](https://user-images.githubusercontent.com/50059322/183309413-aec25d2b-a25a-4bbd-a7f4-de3496a20ad9.png)

Yes but not in the category list with the beautiful little icon :smile:

**Comment by lunastrans** [comment]
I like the way Magic Earth does it. There's still a minimal amount of categories but you can sort them after you click on them. 
![IMG_20220816_144722.jpg](https://user-images.githubusercontent.com/92762322/184883796-a96062fe-fe58-4fed-ba37-e741bd1ee9e0.jpg)
![IMG_20220816_144841.jpg](https://user-images.githubusercontent.com/92762322/184883837-1dc4bc48-64bd-4c58-a5ad-a5357f10bf57.jpg)
![IMG_20220816_144929.jpg](https://user-images.githubusercontent.com/92762322/184883871-54f9f0da-b591-480b-8f02-cfea17afffaa.jpg)

**Comment by biodranik** [comment]
Popular categories should be saved in the search history, no?

**Comment by Altonss** [comment]
> Popular categories should be saved in the search history, no?

Yes that's true, but it's not as intuitive as going to the categories list and finding the category (and the icon) and suits your need. When searching for a category and its never clearly displayed that it is a category search. On the map it then appears clearly, but not in the search bar. New users might even not know you can search for other categories. So maybe it would be appropriate to add an icon (or any other visual marker/explanation) next to the category when distinguising between category and non-category results in the search.
But nonetheless I think adding "Bakery" in the default Categories list would be a logical moove. Appart from this specific category the category search could be made more intuitive and visible to the user :)
